ui-tests/src/test/java/org/openmrs/reference/page/ManageFormsPage.java

```diff
@@ -22,7 +22,7 @@
     private static String REST_PATH_2 = "'\"]";
     private static By CALENDAR_EYE_FORM = By.id("w1-display");
     private static By DATAPICKER_YEAR = By.className("ui-datepicker-year");
-    private static By DATAPICKER_MONTH = By.className("ui-datepicker-moth");
+    private static By DATAPICKER_MONTH = By.className("ui-datepicker-month");
```

Comment:
are you sure this class is renamed in the hosting module? if not you can have this refactoring/renaming change in a separate ticket such that it's all tracked at once

Reply:
sure this is a typo error, Last meeting on TAC  cal we discussed of resolving these bugs along uitest automatation Test

Reply:
Pr to fix the error logged in here https://github.com/openmrs/openmrs-distro-referenceapplication/pull/359

Reply:
Hello @k-joseph  can we merge this  https://github.com/openmrs/openmrs-distro-referenceapplication/pull/359 such that i can work on the updated and clean pull request , could you have a look